src/utilities/renderer-helper.ts

```diff
@@ -131,29 +138,29 @@ export const rendererResponseValidationScheme = Joi.object({
     })).required(),
     debug: Joi.object({
         // TODO are these required or optional
-        debug: Joi.array().items(Joi.string()).required(),
-        internal: Joi.array().items(Joi.string()).required(),
-        perl_warn: Joi.string().allow('').required(),
-        pg_warn: Joi.array().items(Joi.string()).required(),
-        render_warn: Joi.array().items(Joi.string()).optional(), // THIS FIELD IS NEW, replace with required 
+        // debug: Joi.array().items(Joi.string()).required(),
```

Comment:
@doombot-exe we should sync on this, I had issues where it said it wasn't an array when I used the preview with the source

(but I don't think this happened with the renderer helper)